controllers/argo_cluster.go

```diff
@@ -48,8 +53,8 @@ type ArgoTLS struct {
 // NewArgoCluster return a new ArgoCluster
 func NewArgoCluster(c *CapiCluster, s *corev1.Secret) *ArgoCluster {
 	return &ArgoCluster{
-		NamespacedName: BuildNamespacedName(s.ObjectMeta.Name),
-		ClusterName:    c.Clusters[0].Name,
+		NamespacedName: BuildNamespacedName(s.ObjectMeta.Name, s.ObjectMeta.Namespace),
+		ClusterName:    prependNamespace(c.Clusters[0].Name, s.ObjectMeta.Namespace),
```

Comment:
nit: maybe for consistency, switch this name to `BuildClusterName`? wdyt?

Reply:
yep, that makes sense. will change it.